feature_engine/variable_manipulation.py

```diff
@@ -9,7 +9,7 @@
 
 def _define_variables(
     variables: Union[str, Optional[List[str]]]
-) -> Optional[List[str]]:
+) -> Union[str, Optional[List[str]]]:
```

Comment:
@ChristopherGS but the problem here is that this function does not return a string. It returns a list or None, That's why I don't understand the mypy error.